e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-no-companies-house-link.spec.js

```diff
@@ -0,0 +1,75 @@
+import { companyDetails } from '../../../../pages/your-business';
+import partials from '../../../../partials';
+import {
+  cannotApplyPage,
+} from '../../../../pages/shared';
+import { PAGES } from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.APPLY_OFFLINE;
+
+const {
+  ROOT,
+  APPLY_OFFLINE,
+  EXPORTER_BUSINESS: {
+    COMPANY_DETAILS,
+    NO_COMPANIES_HOUSE_NUMBER,
+  },
+} = ROUTES.INSURANCE;
+
+context('Insurance - Your business - Company details page - As an Exporter it should take me to the apply offline page if I do not have a companies house number', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const url = `${Cypress.config('baseUrl')}${ROOT}/${referenceNumber}${COMPANY_DETAILS}`;
+
+      cy.visit(url, {
+        auth: {
+          username: Cypress.config('basicAuthKey'),
+          password: Cypress.config('basicAuthSecret'),
+        },
+      });
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  it('should redirect to APPLY_OFFLINE page when pressing the no companies house number link', () => {
```

Comment:
`${APPLY_OFFLINE}` please

Reply:
There is no `APPLY_OFFLINE` variable in the code though?

Reply:
There is, it's used on line 59 😛

Reply:
done